src/lib/run-studies.ts

```diff
@@ -1,10 +1,33 @@
 /* eslint-disable no-console */
 import { ECSClient, RunTaskCommandOutput } from '@aws-sdk/client-ecs'
 import { ResourceGroupsTaggingAPIClient } from '@aws-sdk/client-resource-groups-tagging-api'
-import { getECSTaskDefinition, registerECSTaskDefinition, runECSFargateTask, getTaskResourcesByRunId } from './aws'
-import { managementAppGetRunnableStudiesRequest, toaGetRunsRequest, filterManagmentAppRuns } from './utils'
+import {
+    getECSTaskDefinition,
+    registerECSTaskDefinition,
+    runECSFargateTask,
+    getTaskResourcesByRunId,
+    RUN_ID_TAG_KEY,
+    TITLE_TAG_KEY,
+    getTaskDefinitionsWithRunId,
+    deleteECSTaskDefinitions,
+} from './aws'
+import {
+    managementAppGetRunnableStudiesRequest,
+    toaGetRunsRequest,
+    filterManagmentAppRuns,
+    filterOrphanTaskDefinitions,
+} from './utils'
 import 'dotenv/config'
 
+const ecsClient = new ECSClient()
+const taggingClient = new ResourceGroupsTaggingAPIClient()
+
+// Set in IaC
+const cluster = process.env.ECS_CLUSTER || ''
+const baseTaskDefinition = process.env.BASE_TASK_DEFINITION_FAMILY || ''
+const subnets = process.env.VPC_SUBNETS || ''
+const securityGroup = process.env.SECURITY_GROUP || ''
```

Comment:
Yeah, it does since they are referenced within the same file / module. 

However, for cleanliness / to avoid confusion I moved these into `runStudies()` itself. Doing that actually uncovered a little wart where in `launchStudy` we were accessing `ecsClient` instead of the `client` passed as an argument :sweat_smile: , so [this change](https://github.com/safeinsights/setup-app/pull/10/commits/364746af1ff46c6def538159f1d98c226626b297) will probably help avoid that type of thing in general.